packages/jest-runtime/src/script_transformer.js

```diff
@@ -412,6 +416,20 @@ const writeCacheFile = (cachePath: Path, fileData: string) => {
   }
 };
 
+/**
+ * On Windows, renames are not atomic, leading to EPERM exceptions when two
+ * processes attempt to rename to the same target file at the same time.
+ * If the target file exists we can be reasonably sure another process has
+ * legitimately won a cache write race and ignore the error.
```

Comment:
Can you mention #4444 in the comment so that we'll hopefully come back later and revert this when it's not necessary?